go_router/test/go_router_test.dart

```diff
@@ -687,13 +658,10 @@ void main() {
           ],
         ),
       ];
-      try {
+      expect(() {
         final router = _router(routes);
         router.goNamed('person', params: {'fid': 'f2', 'pid': 'p1'});
-      } on Exception catch (ex) {
-        dump(ex);
-        assert(false, true);
-      }
+      }, throwsException);
```

Comment:
@csells is this correct? This test is failing

Reply:
I'll take a look

Reply:
this test was throwing an exception which I was dumping while I was developing the fix. it shouldn't throw now and it doesn't.